File: backend/src/main/java/com/example/bbmovie/service/payment/dto/PaymentRequest.java
package com.example.bbmovie.service.payment.dto;

import lombok.*;

import java.math.BigDecimal;

@Data
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class PaymentRequest {
    private String paymentMethodId;
    private BigDecimal amount;
    private String currency;
    private String userId;
    private String orderId;
}

File: backend/payment/src/main/kotlin/com/bbmovie/payment/service/payment/dto/PaymentRequest.kt
package com.bbmovie.payment.service.payment.dto

import java.math.BigDecimal

data class PaymentRequest(
    var paymentMethodId: String? = null,
    var amount: BigDecimal? = null,
    var currency: String? = null,
    var userId: String? = null,
    var orderId: String? = null
)
